VersionMiner/Services/IArgParsingService.cs

```diff
@@ -0,0 +1,24 @@
+// <copyright file="IArgParsingService.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace VersionMiner.Services;
+
+/// <summary>
+/// Parses arguments and responds based on successful or unsuccessful parsing.
```

Comment:
Wordy
```suggestion
/// Parses arguments and responds.
```

Reply:
I refactored to below.  Let me know if this is ok.

```suggestion
/// Parses arguments and responds with success or failure.
```